package.json

```
@@ -26,7 +26,8 @@
     "docker-stop:dev": "docker-compose -f docker-compose.dev.yaml stop",
     "chain-setup": "tsx ./test-setup/main.ts",
     "clean": "rm -Rf dist",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "lint": "eslint \"{apps,libs}/**/*.ts\"",
+    "lint:fix": "eslint \"{apps,libs}/**/*.ts\" --fix",
```

I think we've been trending toward format for this
```suggestion
    "format": "eslint \"{apps,libs}/**/*.ts\" --fix",
```

---

(assuming we implement `prettier` separately at some point in this repo...) does `eslint --fix` fix any lint errors besides formatting? ie, once we have `prettier`, can we replace with just `prettier --write`, or should we still invoke `eslint --fix` as well? And then is that really a "format" target, or is "fix" more appropriate? Again, we should standardize at some point...

---

I think everywhere we have `format` it does both eslint and prettier. That said, that is more from the age that eslint did a bit of both. I view eslint --fix as still a formatter in how it alters code (never changing logic, but changing the structure and layout)